pkgs/games/minetest/default.nix

```diff
@@ -29,37 +27,33 @@
 , postgresql
 , hiredis
 , libiconv
-, zlib
-, libXrandr
-, libX11
 , ninja
 , prometheus-cpp
-, mesa
 , OpenGL
 , OpenAL ? openal
 , Carbon
 , Cocoa
-, withTouchSupport ? false
+, Kernel
 , buildClient ? true
 , buildServer ? true
 }:
 
```

Comment:
Optionally, change all those Darwin dependencies by Darwin and use 
```nix
  let
    inherit (darwin.apple_sdk.frameworks) OpenGL OpenAL Carbon Cocoa Kernel;
  in
  . . . 
```

Reply:
That’s worse for splicing and pointless churn because the SDK refactor will make those dependencies unnecessary by 24.11.

Reply:
> That’s worse for splicing

AFAICR it is exactly the opposite.

The title of that dreadful issue is, precisely, _Discourage/prohibit inheriting from package sets in callPackage_ - and the example given was precisely Darwin Apple SDK: https://github.com/NixOS/nixpkgs/issues/204303

However, [as it was said before](https://discourse.nixos.org/t/frustrations-about-splicing/49607), splicing is dark magic.
I prefer to [see dark magicians fighting one against other about splicing-of-the-gaps](https://github.com/NixOS/nixpkgs/pull/309064#discussion_r1617172095) instead of being in the middle of this crossfire.